### ISSUE TITLE
MinGW 10 Release Announcement

### ISSUE THREAD

**Issue by bgeller** [copperspice/copperspice#257]
We have some very exciting news to share. After a lot of research and the support of a few developers with the LLVM project we have figured out how to cross compile MinGW in a Docker container. 

As of today we are releasing a version of **MinGW 10** that was built directly from source. In addition most of the necessary POSIX utilities have been built using the upstream source code. 

Our fundamental requirement was to build directly from source and not use Msys2 or Cygwin. This has been achieved and we are extremely happy. 

https://github.com/copperspice/cs_mingw

**Comment by bgeller** [comment]
We were asked to modify the git repo name by the MinGW-64 project. The name has been changed from `cs_mingw` to `cs_mingw_w64`

https://github.com/copperspice/cs_mingw_w64

**Comment by 97tn** [comment]
Is the next release of CopperSpice going to be built using this?

**Comment by bgeller** [comment]
Yes, the next 64-bit MinGW release will be built using our new tool chain.  The following page in our CS Overview documentation shows all of our supported platforms and the corresponding compiler versions. 

https://www.copperspice.com/docs/cs_overview/supported-platforms.html
